.gitignore

```diff
@@ -37,3 +37,5 @@ kuma/static/js/libs/ckeditor/source/ckbuilder
 htmlcov/
 locale/**/*.mo
 /bower_components
+kuma/static/compiled-css/*
```

Comment:
Okay, sure thing. Is there a different top-level directory you would recommend, or would we prefer to create something like a */compiled-assets* directory?